scrapinghub/client.py

```diff
@@ -305,6 +306,16 @@ def __init__(self, client, projectid, spiderid, spidername):
         self.jobs = Jobs(client, projectid, self)
         self._client = client
 
+    @wrap_http_errors
+    def update_tags(self, add=None, remove=None):
+        params = get_tags_for_update(add=add, remove=remove)
+        if not params:
+            return
+        path = 'v2/projects/{}/spiders/{}/tags'.format(self.projectid, self.id)
+        url = urljoin(self._client._connection.url, path)
+        response = self._client._connection._session.patch(url, json=params)
```

Comment:
@chekunkov I can't reuse existing `Connection` class in a better way, as there's no PATCH request type there and adding it doesn't look like a good idea. Adding similar system (`_get`/`_post`,`_build_url`) to the new `ScrapinghubClient` class also doesn't look appropriate now, do you agree that using something simple here makes sense for now? Later when we add some new functionality, we can move the part into a separate function(s).